site/pages/competencies/engineer.html

```diff
@@ -0,0 +1,11 @@
+---
+title: Mid-level Engineer Competencies
+description: Engineering competencies for mid-level Engineers.
+permalink: /competencies/engineer/
+layout: o-layout-docs
```

Comment:
All of these new competency pages will need a redirect from the old URL I think, just in case people have it bookmarked etc. Jekyll allows this with [a plugin](https://github.com/jekyll/jekyll-redirect-from) that's enabled by default on GitHub Pages.

You'll need to add some front-matter to each of the pages, so for this one it'd be:

```suggestion
layout: o-layout-docs
redirect_from:
  - /competencies/junior-to-mid/
```

Reply:
oh, good thought!